vitrina/orgs/views.py

```diff
@@ -54,3 +65,58 @@ def get_context_data(self, **kwargs):
         organization: Organization = self.object
         context_data['members'] = organization.representative_set.all()
         return context_data
+
+
+class RepresentativeCreateView(LoginRequiredMixin, PermissionRequiredMixin, CreateView):
+    model = Representative
+    form_class = RepresentativeCreateForm
+    template_name = 'base_form.html'
+
+    def has_permission(self):
+        return self.request.user.role == Representative.COORDINATOR
+
+    def form_valid(self, form):
+        self.object: Representative = form.save(commit=False)
+        organization_id = self.kwargs.get('organization_id')
+        organization = get_object_or_404(Organization, pk=organization_id)
+        self.object.organization = organization
+        try:
+            user = User.objects.get(email=self.object.email)
+            user.organization = organization
+            user.role = self.object.role
+            user.save()
+
+            self.object.user = user
+            self.object.first_name = user.first_name
+            self.object.last_name = user.last_name
+            self.object.phone = user.phone
+        except ObjectDoesNotExist:
+            send_mail(
+                subject=_('Užsiregistruokite'),
+                message=_('Prašome užsiregistruoti adresu https://data.gov.lt/register'),
```

Comment:
Do not hardcode `data.gov.lt` use sites functionality, this was used somewhere already. And link should go to a registration, after which, user will be added as a member. So here we need to pass some kind of token, that will be validated and after registration, user will be assigned as member of given organization.